ExampleMod/Common/CustomEquipmentSlot/EarringsGlobalItem.cs

```diff
@@ -0,0 +1,35 @@
+using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.CustomEquipmentSlot
+{
+	// Note: To fully understand this example, please start by reading https://github.com/tModLoader/tModLoader/tree/stable/ExampleMod/Common/CustomEquipmentSlot/README.md
```

Comment:
I think that this should not be called an 'equipment slot' perhaps `CustomAccessoryVisual`